arviz/tests/test_data.py

```diff
@@ -431,15 +442,15 @@ def test_inference_data2(self):
             assert hasattr(inference_data, 'posterior_predictive')
             assert hasattr(inference_data, 'observed_data')
 
-    def test_inference_data3(self):
+    def test_inference_data_input_types2(self):
```

Comment:
Not required. For these tests a single line comment would be nice explaining the difference between this one and the other 3 similarly named tests. Makes it easier to figure out what the intent was for each one

Reply:
True. I can add them later.